config/300-trigger.yaml

```diff
@@ -71,6 +71,9 @@ spec:
                       type: string
                     source:
                       type: string
+                attributes:
+                  type: object
```

Comment:
Can this be further constrained to a map[string]string? From https://swagger.io/docs/specification/data-models/dictionaries/, I think something like:

```yaml
additionalProperties:
  type: string
```

Reply:
Done, thanks!